src/LayerBrushes/Artemis.Plugins.LayerBrushes.Ambilight/UI/CapturePropertiesViewModel.cs

```diff
@@ -86,6 +87,7 @@ public DisplayPreview SelectedDisplay
             get => _selectedDisplay;
             set
             {
+                if (_closing) return; // Don't update display and region if dialog is being closed to avoid 0 values
```

Comment:
Here we can probably just check `if (!IsActive) return;` without needing our own `_closing`